pkg/config/config.go

```diff
@@ -293,6 +294,27 @@ type (
 		SendSMS                     hookConfig `toml:"send_sms"`
 		SendEmail                   hookConfig `toml:"send_email"`
 	}
+	factorTypeConfiguration struct {
+		EnrollEnabled bool `toml:"enroll_enabled"`
+		VerifyEnabled bool `toml:"verify_enabled"`
+	}
+	totpFactorTypeConfiguration struct {
+		factorTypeConfiguration
+	}
+
+	phoneFactorTypeConfiguration struct {
+		factorTypeConfiguration
+		OtpLength    int           `toml:"otp_length"`
+		Template     string        `toml:"template"`
+		MaxFrequency time.Duration `toml:"max_frequency"`
+	}
+
+	mfa struct {
+		TOTP  totpFactorTypeConfiguration  `toml:"totp"`
+		Phone phoneFactorTypeConfiguration `toml:"phone"`
+		// set to float64 for backward compatibility
+		MaxEnrolledFactors float64 `toml:"max_enrolled_factors"`
```

Comment:
Do we need to consider backwards compatibility if it's a new config? probably use `uint` or the correct type.

Reply:
Yeah good point - the CLI config doesn't need to be compatible against the underlying env config. Let's change